scripts/bootstrap/bootstrap

```diff
@@ -26,8 +26,26 @@ set -eE
 # Path where the actual Entware distribution resides (will be mounted to /opt)
 [[ -z $entware_path ]] && entware_path=/home/root/.entware
 
+# Find out used branch or get default one
+get-branch() {
```

Comment:
Is there a more clean way to do this? Didn't found a good way around defining this function in the env defaults section.

Reply:
You can probably parse out the branch with something like:
```bash
get-branch(){
  local opkg_conf=$entware_path/etc/opkg.conf
  branch=$(cat "$opkg_conf" | grep src/gz | while read -r line; do
    if [[ "$(echo "$line" | awk '{print $2}')" == "toltec" ]];then
      local url=$(echo "$line" | awk '{print $3}')
      if ! echo $url | grep -q https://toltec-dev.org/; then
        continue
      fi
      echo "$url" | sed 's|https://toltec-dev.org/||'
      return
    fi
  done)
  [[ "$branch" != "" ]] && echo "$branch" || echo stable
    
}
```
This will allow for more than just stable and testing if we ever add a new branch. You could also add some extra logic to it to detect that there is a toltec repo added with a completely wrong URL if you'd like.